lab1/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="app.css">
+        
+    </head>
+    <body>
+        <h1>{Incremental Game}</h1>
+        <textarea background="black" style="margin: 0px; width: 1328px; height: 170px;">            
+                {Story line...}
+                {Story line 2 ...}
+                {More story line to come...}
+        </textarea>
```

Comment:
textarea isn't meant for displaying data, should use other tag for displaying data and use `overflow: auto` to create scroll bar